src/runtime/offers-flow.js

```diff
@@ -165,7 +175,9 @@ export class SubscribeOptionFlow {
    */
   maybeOpenOffersFlow_(data) {
     if (data && data['subscribe']) {
-      new OffersFlow(this.deps_, this.options_).start();
+      const options = this.options_ || {};
+      options.isClosable = this.isOffersViewClosable_;
```

Comment:
Are you sure this is needed? Wouldn't the right value already be in the `this.options_`?

Reply:
Added a check and setting the default value only if `isClosable` is not available in options.